apps/privacy-panel/js/tc/applications.js

```diff
@@ -44,7 +46,13 @@ function(panels, appList, appDetails) {
       // some apps might have a localized name in their manifest
       window.addEventListener('localized', refreshAppList);
 
-      appList.init(permissionTable).then(this.renderAppList.bind(this),
+      var appSearch = document.getElementById('tc-appSearch');
+      appSearch.onsubmit = () => this._closeSearchBox();
```

Comment:
I’ve taken the “search” field from the Contacts app to match DT’s UX… and surprisingly, in this app the “Cancel” button has a “submit” type. Starting over.